selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/stack/CreateWorkspaceFromCentosNodeStackTest.java

```diff
@@ -0,0 +1,119 @@
+  private static final String WEB_NODEJS_SIMPLE = "web-nodejs-simple";
+
+  private List<String> projects =
+      ImmutableList.of(ANGULAR_PATTERNFLY_STARTER, NODEJS_HELLO_WORLD, WEB_NODEJS_SIMPLE);
+
+  @Inject private Ide ide;
+  @Inject private Consoles consoles;
+  @Inject private Dashboard dashboard;
+  @Inject private DefaultTestUser defaultTestUser;
+  @Inject private ProjectExplorer projectExplorer;
+  @Inject private CreateWorkspaceHelper createWorkspaceHelper;
+  @Inject private TestWorkspaceServiceClient workspaceServiceClient;
+
+  @BeforeClass
+  public void setUp() {
+    dashboard.open();
+  }
+
+  @AfterClass
+  public void tearDown() throws Exception {
+    workspaceServiceClient.delete(WORKSPACE_NAME, defaultTestUser.getName());
+  }
+
+  @Test
+  public void checkWorkspaceCreationFromCentosNodeStack() {
+    createWorkspaceHelper.createWorkspaceFromStackWithProjects(
+        CENTOS_NODEJS, WORKSPACE_NAME, projects);
+
+    ide.switchToIdeAndWaitWorkspaceIsReadyToUse();
+
+    projectExplorer.waitProjectInitialization(ANGULAR_PATTERNFLY_STARTER);
+    projectExplorer.waitProjectInitialization(NODEJS_HELLO_WORLD);
+    projectExplorer.waitProjectInitialization(WEB_NODEJS_SIMPLE);
+  }
+
+  @Test(priority = 1)
+  public void checkAngularPatternfyStarterProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
+        ANGULAR_PATTERNFLY_STARTER,
+        BUILD_GOAL,
+        INSTALL_DEPENDENCIES_COMMAND_ITEM.getItem(ANGULAR_PATTERNFLY_STARTER),
+        "bower_components/font-awesome");
+    consoles.executeCommandFromProjectExplorer(
+        ANGULAR_PATTERNFLY_STARTER,
+        RUN_GOAL,
+        RUN_COMMAND_ITEM.getItem(ANGULAR_PATTERNFLY_STARTER),
+        "Waiting...");
+    consoles.checkWebElementVisibilityAtPreviewPage(By.xpath("//*[@id='pf-app']"));
+    consoles.closeProcessTabWithAskDialog("angular-patternfly-starter:run");
+  }
+
+  @Test(priority = 1)
+  public void checkNodejsHelloWorldProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
+        NODEJS_HELLO_WORLD,
+        RUN_GOAL,
+        RUN_COMMAND_ITEM.getItem(NODEJS_HELLO_WORLD),
+        "Example app listening on port 3000!");
+    consoles.checkWebElementVisibilityAtPreviewPage(By.xpath("//*[text()='Hello World!']"));
+    consoles.closeProcessTabWithAskDialog(RUN_COMMAND_ITEM.getItem(NODEJS_HELLO_WORLD));
+  }
+
+  @Test(priority = 1)
+  public void checkWebNodejsSimpleProjectCommands() {
+    consoles.executeCommandFromProjectExplorer(
+        WEB_NODEJS_SIMPLE,
+        BUILD_GOAL,
+        INSTALL_DEPENDENCIES_COMMAND_ITEM.getItem(WEB_NODEJS_SIMPLE),
+        "bower_components/angular");
+    consoles.executeCommandFromProjectExplorer(
+        WEB_NODEJS_SIMPLE,
+        RUN_GOAL,
+        RUN_COMMAND_ITEM.getItem(WEB_NODEJS_SIMPLE),
+        "Started connect web server");
+    consoles.checkWebElementVisibilityAtPreviewPage(
+        By.xpath("//p[text()=' from the Yeoman team']"));
```

Comment:
It's not quite clear what "//p[text()=' from the Yeoman team']" is about. From this point it would be better to name the string somehow.